src/Db/Query/Where/Where.php

```diff
@@ -138,6 +139,10 @@ public function toStringWithoutLogic(IQuery $query): string
         {
             case 'between':
             case 'not between':
+                if (!\is_array($thisValues) || !isset($thisValues[0],$thisValues[1]))
+                {
+                    throw new \RuntimeException(sprintf('where %s value must be [beginValue, endValue]', $operation));
+                }
```

Comment:
```suggestion
                if (!\is_array($thisValues) || !isset($thisValues[0], $thisValues[1]))
                {
                    throw new \InvalidArgumentException(sprintf('where %s value must be [beginValue, endValue]', $operation));
                }
```

参数少了个空格，cs没开这个规则？
感觉异常可以换个更明确的`InvalidArgumentException`